Detectors/MUON/MCH/IO/src/digits-writer-workflow.cxx

```diff
@@ -56,9 +56,13 @@ void customize(std::vector<ConfigParamSpec>& workflowOptions)
 WorkflowSpec defineDataProcessing(const ConfigContext& cc)
 {
   bool useMC = cc.options().get<bool>("enable-mc");
+  auto name = fmt::format("mch-digit-writer-{}-{}",
+                          cc.options().get<std::string>("input-digits-data-description"),
+                          cc.options().get<std::string>("input-digits-data-description"));
+
```

Comment:
I imagine the second argument is supposed to be "input-digitrofs-data-description" ?

Reply:
@aphecetche , could you comment? I added here your patch, which worked for me. But indeed, it seems that @pillot might have a point.
Cheers,
Chiara

Reply:
Philippe indeed has a point. It's still working as the only thing that matters is that the name is different in the two digit writers used, and they are. But better to correct it yes.

Reply:
Done!